pages.de/common/ab.md

```diff
@@ -21,4 +21,4 @@
 
 - Sende 100 HTTP POST Anfragen an eine gegebene URL, wobei eine JSON Belastung aus einer Datei verwendet wird:
 
-`ab -n {{100}} -T {{application/json}} -p {{daten.json}} {{url}}`
+`ab -n {{100}} -T {{application/json}} -p {{pfad/zur/datei.json}} {{url}}`
```

Comment:
I'd prefer `pfad/zu/datei.json`, since otherwise it would be `path/to/the/file.json`.

Reply:
Okay, thank you very much!! I didn't think of asking you.  I just copied it from another file.  We should probably change all of these too: https://github.com/tldr-pages/tldr/search?q=pfad%2Fzur%2Fdatei, don't you think?

Reply:
Yeah, the paths in the german files are very inconsistent. I'll have a look at that.

Reply:
@bl-ue there's actually a lot more to fix. Can I combine all that to one PR?